File: tests/Generator/resource/csharp/complex/Specification.cs
using System.Text.Json.Serialization;

public class Specification
{
    [JsonPropertyName("import")]
    public System.Collections.Generic.Dictionary<string, string>? Import { get; set; }

    [JsonPropertyName("definitions")]
    public System.Collections.Generic.Dictionary<string, DefinitionType>? Definitions { get; set; }

    [JsonPropertyName("root")]
    public string? Root { get; set; }

}

File: tests/Generator/resource/rust/complex/specification.rs
use serde::{Serialize, Deserialize};
use definition_type::DefinitionType;

#[derive(Serialize, Deserialize)]
pub struct Specification {
    #[serde(rename = "import")]
    import: Option<HashMap<String, String>>,

    #[serde(rename = "definitions")]
    definitions: Option<HashMap<String, DefinitionType>>,

    #[serde(rename = "root")]
    root: Option<String>,

}

